samples/test/features/security-questions.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Security Questions
+
+  Background:
+    Given a Policy that defines "Profile Enrollment"
+      And with a Policy Rule that defines 'collecting default attributes and a required "Age" and emailVerification is not required'
```

Comment:
`age` is a custom attr and is irrelevant with security question authenticator, it can be ignored here.